lib/msal-core/package.json

```diff
@@ -43,11 +43,11 @@
   "devDependencies": {
     "@types/chai": "^4.1.7",
     "@types/handlebars": "4.0.33",
+    "@types/js-base64": "^2.3.1",
```

Comment:
Any specific reason for the change of order?

Reply:
`npm` puts dependencies in alphabetical order (they were not before).